internal/registry/registry.go

```diff
@@ -25,8 +25,10 @@ import (
 )
 
 const (
-	// providerKeyPath is where provider info is stored in to indexer repo
+	// providerKeyPath is where provider info is stored in to indexer repo.
 	providerKeyPath = "/registry/pinfo"
+	// pollCheckInterval is how often to check for idle providers to poll.
+	pollCheckInterval = time.Hour
```

Comment:
why is this a constant? can we make it a config param as well?

Reply:
I changed this to use `cfg.PollRetryAfter` as the check interval, becuase that will be the lowest frequency the check can be run and still be able to retry to sync in the configured time.